src/signals/settings/categories/components/AddIconExplanation.tsx

```diff
@@ -0,0 +1,57 @@
+// SPDX-License-Identifier: MPL-2.0
```

Comment:
Personally think AddIcon is a strange name. Wouldn't UploadIconExplanation be a better one?

Reply:
I personally am the worst when it comes to names. Too long, not on point etc etc. I've changed it.